### PR TITLE
Refs: #5: Setup UI and logic for forgot password  restaurant side

### LINKED ISSUE #5: Forgot Password UI for restaurant side
Due day Tuesday, January 30
- [ ] Create a UI for reset password page for restaurant side ( quality needs to match figma file)

### PR DESCRIPTION
**Pull request by Silverback2023**
- followed Figma design
- components derived from material ui
- check and confirm design 
- check readiness for Firebase integration

### CODE REVIEW

File: src/restaurantSide/pages/auth/ForgotPasswordHost.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      severity: 'success',
+      message: 'Password reset email sent. Check your inbox.',
+    });
+  };
+
+  const handleCloseNotification = () => {
+    setNotification({ ...notification, open: false });
+  };
+
+  const [isFocused] = useState(false);
```
Comment (PhucMaii) [inline src/restaurantSide/pages/auth/ForgotPasswordHost.jsx]: You should move this one to the useState section

File: src/restaurantSide/pages/auth/ForgotPasswordHost.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+  return (
+    <Grid container justifyContent='center' alignItems='center' spacing={2} style={{ minHeight: '100vh', maxWidth: '1500px' }}>
```
Comment (PhucMaii) [inline src/restaurantSide/pages/auth/ForgotPasswordHost.jsx]: You shouldn't add fixed pixels width to the container, try to figure other ways

File: src/restaurantSide/pages/auth/ForgotPasswordHost.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  Snackbar,
+  Alert,
+  IconButton,
+} from '@mui/material';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import { LogoImg } from './styled';
+
+const ForgotPasswordHost = () => {
+  const [email, setEmail] = useState('');
+  const [notification, setNotification] = useState({
```
Comment (PhucMaii) [inline src/restaurantSide/pages/auth/ForgotPasswordHost.jsx:19]: Good work

File: src/restaurantSide/pages/auth/ForgotPasswordHost.jsx (hunk 1 above)
Comment (Silverback2023) [reply src/restaurantSide/pages/auth/ForgotPasswordHost.jsx]: Done

File: src/restaurantSide/pages/auth/ForgotPasswordHost.jsx (hunk 2 above)
Comment (Silverback2023) [reply src/restaurantSide/pages/auth/ForgotPasswordHost.jsx]: Did this: <Grid container justifyContent='center' alignItems='center' style={{ minHeight: '100vh' }}>
      {}
      <Grid item xs={12} sm={10} md={8} lg={6} xl={4}>

**Comment by Silverback2023** [comment]
- fixed screen responsiveness
- changed link address name to forgot-password
- items alignment fixed

**Pull request merged**